graph/GuidelinesGraph.md

```diff
@@ -330,6 +319,136 @@ For a complete mapping of error codes to HTTP statuses, see
 
 <a name="api-contract-and-non-backward-compatible-changes"></a>
 
+### Enums
+
+In OData, enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
```

Comment:
@XVincentX It seems that the topic of Enums is a bit out of context and sequence as it placed right now. We clearly need a description of when to use enums for modelling, but it falls under the 'Resource modelling' section in the main guidelines IMO. I would also make the Enum page a separate page and link to the 'Resource modelling' section.